src/lib/socket/server.ts

```diff
@@ -1,14 +1,29 @@
 import { Server } from 'http';
 
+//import { isPayToScriptHash } from '@bitauth/libauth';
```

Comment:
I commented out those two imports because they weren't in use when I created the branch. Was there a reason for those libraries to be imported in the first place?

Reply:
> I commented out those two imports because they weren't in use when I created the branch. Was there a reason for those libraries to be imported in the first place?

Oh that must've been my editor that added them automatically. My bad! You can go ahead and get rid  of them.